components/header-search-box/header-search-box.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { ReactElement } from 'react';
+import { SearchBox } from 'react-instantsearch-dom';
+
+const HeaderSearchBox = ({placeholderText}: {placeholderText: string}): ReactElement => {
+  return (
+    <>
+      <SearchBox
+        translations={{
+          placeholder: placeholderText,
+        }}
+        submit={
+          <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 18 18">
+            <g
+              fill="none"
+              fillRule="evenodd"
+              stroke="currentColor"
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              strokeWidth="1.67"
+              transform="translate(1 1)"
+            >
+              <circle cx="7.11" cy="7.11" r="7.11" />
+              <path d="M16 16l-3.87-3.87" />
+            </g>
+          </svg>
+        }
+      />
+      <style jsx global>{`
+        .ais-SearchBox-form {
+          position: relative;
+        }
+
+        .ais-SearchBox {
+          height: 57px;
+          width: 460px;
+        }
+
+        .ais-SearchBox-input {
+          height: 57px;
+          background-color: #4a515a;
+          color: #ccc;
```

Comment:
I think we can use name colours here from `colors.ts`

Reply:
Updated.